ppr-ui/src/utils/validators/mhr-registration-validation.ts

```diff
@@ -0,0 +1,16 @@
+import { MhrRegistrationFieldValidationIF } from '@/interfaces/validation-interfaces'
+
+export const mhrRegistrationValidation = {
+  // Form fields and their respective validation rules
+  fields: [
+    {
+      id: 'otherRemarks',
+      validations: (fieldValue: string) => {
+        // All validations would go here, returning the error for each rule
+        if (fieldValue?.length > 140) {
+          return 'Maximum 140 characters'
+        }
+      }
+    }
+  ] as Array<MhrRegistrationFieldValidationIF>
+}
```

Comment:
I appreciate the approach here but this might add unneeded complexity in my opinion. 

Vuetify components that take input have `:rules` prop which can handle the abstract complexity of applying rules and entering an error state/message on the component. 

I believe this could be simplified with a generic input rule in our `useInputRules` composable, something like..
  
![Screen Shot 2022-06-21 at 9 10 27 PM](https://user-images.githubusercontent.com/53542131/174941598-f5b3ca5d-ba91-4ed5-b2a9-69eb18a87b4d.png)

  And just apply the rule to the input `:rules="textAreaRules"`
  
Could probably be more accurately named but yea, i'm not sure we need this abstract method of fetching validations when it's built into vuetify components. 

Thoughts on this?

Reply:
There are multiple ways we do validation in this app. I will look into vuetify's built-in validation. Thanks!

Reply:
Agree, I like the built-it validation and we should go with that, yea.